app/grandchallenge/challenges/admin.py

```diff
@@ -42,7 +42,10 @@ class ChallengeRequestAdmin(ModelAdmin):
 
     @admin.display(description="Total cost")
     def total_cost(self, obj):
-        return obj.budget.get("Total")
+        if obj.budget:
```

Comment:
This addresses the following issue: https://sentry.io/organizations/grand-challenge/issues/3212784983/?project=303639&query=is%3Aunresolved